packages/neon_framework/packages/neon_storage/lib/src/storage/sqlite_persistence.dart

```diff
@@ -2,17 +2,40 @@ import 'dart:async';
 import 'dart:convert';
 
 import 'package:built_collection/built_collection.dart';
-import 'package:flutter/foundation.dart';
 import 'package:logging/logging.dart';
-import 'package:meta/meta.dart' show internal;
-import 'package:neon_framework/platform.dart';
-import 'package:neon_framework/src/storage/persistence.dart';
-import 'package:path/path.dart' as p;
-import 'package:path_provider/path_provider.dart';
-import 'package:sqflite_common_ffi/sqflite_ffi.dart';
+import 'package:meta/meta.dart';
+import 'package:neon_storage/neon_sqlite.dart';
+import 'package:neon_storage/neon_storage.dart';
+import 'package:sqflite_common/sqlite_api.dart';
 
 final _log = Logger('SQLitePersistence');
 
+final class _SQLiteCachedPersistenceTable extends Table {
+  @override
+  String get name => 'preferences';
+
+  @override
+  int get version => 1;
+
+  @override
+  void onCreate(Batch db, int version) {
+    db.execute('''
+CREATE TABLE  IF NOT EXISTS "preferences" (
+	"prefix"  TEXT NOT NULL,
+	"key"     TEXT NOT NULL,
+	"value"   TEXT NOT NULL,
+	PRIMARY KEY("prefix","key"),
+	UNIQUE("key","prefix")
```

Comment:
```suggestion
	PRIMARY KEY("prefix", "key")
```
PRIMARY KEY is already UNIQUE